sdk/communication/Azure.Communication.CallingServer/src/Models/CallingServerModelFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary>
+    /// Calling server model factory
+    /// </summary>
+    [CodeGenModel("AzureCommunicationServicesModelFactory")]
+    internal partial class CallingServerModelFactory
```

Comment:
this probably should be moved to `.Models` namespace and also should it be public so developers can instantiate the models that do not have public constructor.
CC: @minnieliu

Reply:
Yup, agreed, good catch Reza!

Reply:
Is this a pattern followed by other modalities? So, far we don't seem to have this need for apps

Reply:
Yes. See this for more details: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-mocking